news/2019-09-27-mappers-guild-september-newspost.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: post
+title: Mappers' Guild Quests Completed
+date: 2019-09-24 00:00:00 +0000
+---
+
+The Mappers' Guild has been hard at work creating beatmaps for your favourite Featured Artist tracks. If you haven't been following, come check them out!
+
+![](/wiki/shared/news/banners/mappersguild.png)
+
+Since March 2019, the **[Mappers' Guild](https://mappersguild.com)** has been open to everyone having at least three ranked maps. This has made the Guild very productive in terms of mapping and modding. With the introduction of the quest system, mappers have been more active and interested in mapping Featured Artist content than ever before.
+
+The quests are usually artist-specific, but can sometimes be thematic, such as Vocaloid-themed quests for instance, which mixes several artists in one quest. They usually requires mappers to produce a given amount of maps, with sometimes extra rules that make quests more diverse. 
+
+If you're curious and want to know more about the Guild and its quest system, please visit [the official Mappers' Guild website](https://mappersguild.com)!
+
+There are a lot of completed quests, so here we've picked out the ones involving the most mapsets and mappers. You can find even more Featured Artist content by searching the [beatmap listing](https://osu.ppy.sh/beatmapsets?q=featured%20artist)!
+
+![Cranky header image](https://assets.ppy.sh/artists/23/header.jpg)
```

Comment:
```suggestion
![](https://assets.ppy.sh/artists/23/header.jpg)
```
No need to add a subtitle for hard image links. Not sure how the news parser handles them.

Reply:
these aren't titles, they're alt text. works the same on news as wiki (except for the first header )